rest-api-spec/api/search_template.json

```diff
@@ -48,7 +48,7 @@
         },
         "search_type": {
           "type" : "enum",
-          "options" : ["query_then_fetch", "query_and_fetch", "dfs_query_then_fetch", "dfs_query_and_fetch", "count", "scan"],
+          "options" : ["query_then_fetch", "query_and_fetch", "dfs_query_then_fetch", "dfs_query_and_fetch", "scan"],
```

Comment:
I think we have to put this back if we only deprecate `count` instead of removing it? clients will still need to expose it as an option